packages/preferences/src/browser/views/components/preference-select-input.ts

```diff
@@ -38,7 +39,7 @@ export class PreferenceSelectInputRenderer extends PreferenceLeafNodeRenderer<JS
         const values = this.enumValues;
         const defaultValue = this.preferenceNode.preference.data.default;
         for (let i = 0; i < values.length; i++) {
-            const value = `${values[i]}`;
+            const value = escapeInvisibleChars(`${values[i]}`);
             const detail = PreferenceProvider.deepEqual(defaultValue, value) ? 'default' : undefined;
```

Comment:
Suggestion: I believe that this messes with the `deepEqual` check in case `value !== escapeInvisibleChars(value)`. I would recommend the follwing:

```suggestion
            const value = values[i];
            const label = escapeInvisibleChars(`${value}`);
            const detail = PreferenceProvider.deepEqual(defaultValue, value) ? 'default' : undefined;
```

And then use the `label` later at the `items.push` invocation.

Reply:
Thanks for you suggestion, I modified it.